colossus/src/main/scala/colossus/protocols/http/HttpResponse.scala

```diff
@@ -1,77 +1,151 @@
 package colossus
 package protocols.http
 
-import akka.util.ByteString
-import colossus.controller.{FiniteBytePipe, Source}
+import akka.util.{ByteString, ByteStringBuilder}
+import colossus.controller.Source
 import colossus.core.DataBuffer
+import core._
+import service.Callback
+import controller._
+import HttpParse._
+
+
+case class HttpResponseHead(version : HttpVersion, code : HttpCode, headers : Vector[(String, String)] = Vector()) extends HttpHeaderUtils {
```

Comment:
yeah I'm thinking of doing that at the same time we move some of the special headers into separate vals.